LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2017 Daniel Holzmann <d@velopment.at>
+Copyright (c) 2017 Marc Nitzsche <hello@marcnitzsche.de>
```

Comment:
Ehrlich gesagt, keine Ahnung. Vielleicht ist es ein Copyright by X and Y; ich hatte aber einen krassen Refactor gemacht mit API change, von daher.